README.md

```diff
@@ -17,7 +17,7 @@ Manual Installation
 For Ubuntu Server 16.04
 
 ```
-apt install python2.7 python2.7-dev virtualenv gcc pkg-config libpng-dev libjpeg-dev libfreetype6-dev postgresql-server-dev-all libgeos-dev g++ python-shapely nodejs
+apt install python2.7 python2.7-dev virtualenv gcc pkg-config libpng-dev libjpeg-dev libfreetype6-dev postgresql-server-dev-all libgeos-dev g++ python-shapely nodejs npm
```

Comment:
We don't have this package on our infrastructure - is it necessary on recent distrib versions?

Reply:
I didn't check on Debian, but on Ubuntu it is necessary.

Based [on that](https://github.com/osm-fr/osmose-frontend/pull/237#pullrequestreview-392116772) I will update this section.